### PR TITLE
pkg: make build output quiet

### PR DESCRIPTION
**Pull request by toonst**
### Contribution description
Remove git command output while building packages. 

### Testing procedure
Run make in RIOT app which uses packages.

### CODE REVIEW

File: pkg/pkg.mk [hunk 1]
```diff
@@ -21,28 +21,28 @@ git-download: git-ensure-version
...
 	fi
 
 $(PKG_BUILDDIR)/.git-downloaded:
-	rm -Rf $(PKG_BUILDDIR)
-	mkdir -p $(PKG_BUILDDIR)
-	$(GITCACHE) clone "$(PKG_URL)" "$(PKG_VERSION)" "$(PKG_BUILDDIR)"
-	touch $@
+	$(Q)rm -Rf $(PKG_BUILDDIR)
+	$(Q)mkdir -p $(PKG_BUILDDIR)
+	$(Q)$(GITCACHE) clone -q "$(PKG_URL)" "$(PKG_VERSION)" "$(PKG_BUILDDIR)"
```
Comment (aabadie) [inline pkg/pkg.mk]: option `-q` doesn't work with gitcache.
To reproduce: run `make distclean` then try to build again a package (I'm testing with `tests/pkg_cayenne-lpp`).

**Comment by toonst** [comment]
Thanks for the review. I addressed your comments & suggestions.

**Comment by aabadie** [comment]
I tested this PR and works as expected. I like the proposed approach. Maybe @kaspar030 have other comments?

**Comment by toonst** [comment]
I think we can go ahead with this. @aabadie @kaspar030 what do you think?

**Comment by aabadie** [comment]
All green, let's go!

**Pull request merged**